### ISSUE TITLE
API seems to be down since a while and the page defunct

### ISSUE THREAD

**Issue by TrashAccount3424** [OnTheDEX/xrpledger-token-data-api#3]
API seems to be down since a while and the page mostly defunct.
Is this project dead or should we expect it to be fixed?

`{"error": "ERROR_MAINTENANCE", "message": "The system is under maintenance for a short period.  Please try again later."}`

**Comment by OnTheDEX** [comment]
Yes it is down for a moment while it recovers from multiple rippled failures over recent days. Possibly hardware (disk) issues but it’s currently playing catch-up and should be back up later today at some point.  Sorry for any inconvenience.

**Comment by TrashAccount3424** [comment]
> Yes it is down for a moment while it recovers from multiple rippled failures over recent days. Possibly hardware (disk) issues but it’s currently playing catch-up and should be back up later today at some point. Sorry for any inconvenience.

Seems to be broken again ;/

**Comment by TrashAccount3424** [comment]
Seems offline again

**Comment by TrashAccount3424** [comment]
again
